src/components/CardForm.js

```diff
@@ -0,0 +1,65 @@
+import { StyledButton } from './StyledButton';
+import styled from 'styled-components';
+
+export default function CardForm({ onSubmit, cards, onClick }) {
+  const TEXT_MAX_LENGTH = 200;
+  return (
+    <StyledForm onSubmit={handleSubmit} aria-labelledby="create-card" autoComplete="off" name="create">
+      <h2 id="create-card"> {cards.length > 0 ? 'Erstelle eine Karte' : 'Erstelle deine erste Karte'}</h2>
+      <label htmlFor="question">Gib hier deine Frage ein:</label>
+      <input name="question" type="text" placeholder="z.B. Wer hat react entwickelt?" maxLength={TEXT_MAX_LENGTH} id="question" required></input>
```

Comment:
```suggestion
      <input name="question" type="text" placeholder="z.B. Wer hat react entwickelt?" maxLength={TEXT_MAX_LENGTH} id="question" required>
```
input ist nen selfclosing-element

Reply:
> input braucht kein closing tag

Mach ich, aber mit /> am Tag-Ende, danke dir @tRomanowski